integration-tests/pkgJson.spec.js

```diff
@@ -550,37 +505,4 @@ describe('pkgJson', function () {
             });
         });
     });
-
-    // No pkg.json included in test file.
-    describe('local path is added to config.xml without pkg.json', function () {
-        beforeEach(() => useProject('basePkgJson13'));
```

Comment:
Did you check if the fixture is still used now that this has been deleted? What is the new behavior without a package.json?

Reply:
I did not really look into the tests beyond trying to make them pass. 😓 

My hope is to revisit these when we update the package.json handling.

Reply:
Well `basePkgJson13` refers to a project fixture that might be unused now. I can't check it now though.